styles/css/vtex.flex-layout.css

```diff
@@ -43,8 +28,8 @@
 
 .flexCol--itemsSummary,
 .flexCol--itemsSummaryMobile {
-  background-color: #f2f4f5;
   height: 100%;
+  animation: fade-in-right 500ms cubic-bezier(0.455, 0.030, 0.515, 0.955) 480ms both;
```

Comment:
this animation is also happening when the drawer is opened, which I don't think is intended right?

Reply:
Yea! Nice catch